docs/example.rst

```diff
@@ -1,46 +1,102 @@
+movie reviews. For that reason, we'll merge both folders `pos` and `neg`, to one
+named `reviews`:
+
+.. code-block:: shell
+
+    $ mkdir reviews
+    $ mv txt_sentoken/pos/* txt_sentoken/neg/* reviews/
+
+
+High-level interface
+====================
+
+For quick, one-off studies, the command line interface allows you to specify
+minimal information and obtain topic model plot output. For all available
+options, please run ``topik --help``
 
+.. code-block:: bash
 
-For more information on the datasets and download options visit `NLTK data <http://www.nltk.org/data.html>`_.
+    $ topik --help
 
-Instead of using the dataset in for `sentiment analysis`, its initial purpose, we'll perform `topic modeling` on the
-movie reviews. For that reason, we'll merge both folders `pos` and `neg`, to one named `reviews`.
+    Usage: topik [OPTIONS]
 
+      Run topic modeling
 
-High-level interfaces
----------------------
+    Options:
+      -d, --data TEXT        Path to input data for topic modeling  [required]
+      -f, --format TEXT      Data format provided: json_stream, folder_files,
+                             large_json, elastic, solr
+      -m, --model TEXT       Statistical topic model: lda_batch, lda_online
+      -o, --output TEXT      Topic modeling output path
+      -t, --tokenizer TEXT   Tokenize method to use: simple, collocations,
+                             entities, mix
+      -n, --ntopics INTEGER  Number of topics to find
+      --prefix_value TEXT    In 'large json' files, the prefix_value to extract
+                             text from
+      --event_value TEXT     In 'large json' files the event_value to extract text
+                             from
+      --field TEXT           In 'json stream' files, the field to extract text
+                             from
+      --help                 Show this message and exit.
+
+To run this on our movie reviews data set:
+
+.. code-block:: shell
+
+   $ topik -d reviews
+
+The shell command is a front end to :func:`~.run_model`, which is also
+accessible in python:
+
+.. code-block:: python
+
+   >>> from topik.run import run_model
+   >>> run_model("reviews")
 
-As mentioned in the introduction page, there a two high-level interfaces: the command-line interface and the function
-topik.run()
 
 Custom topic modeling flow
---------------------------
+==========================
 
+For interactive exploration and more efficient, involved workflows, there also
+exists a Python API for using each part of the topic modeling workflow. There
+are four phases to topic modeling with topik: data import,
+tokenization/vectorization, modeling and visualization. Each phase is modular, with several
+options available to you for each step.
 
+An example complete workflow would be the following:
 
+.. code-block:: python
 
-Analyzing the results
----------------------
+   >>> from topik import read_input, registered_models
+   >>> raw_data = read_input("reviews")
```

Comment:
No, my humble opinion is that this is truly required.  The probability of failure when this information is not provided or not accurate is practically 100%.  If anything, run_model should add the requirement.